packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -15,49 +15,22 @@
     </oui-message>
 
     <oui-message
-        data-ng-if="$ctrl.users.length === 0 && $ctrl.allowedIps.length > 0"
+        data-ng-if="$ctrl.getWarningMessage()"
         data-type="warning"
         data-dismissable
     >
-        <span
-            data-translate="pci_databases_general_information_no_users"
-        ></span>
-        <a href data-ng-click="$ctrl.manageUsers()">
-            <span
-                data-translate="pci_databases_general_information_manage_users"
-            ></span>
-        </a>
-    </oui-message>
-
-    <oui-message
-        data-ng-if="$ctrl.allowedIps.length === 0 && $ctrl.users.length > 0"
-        data-type="warning"
-        data-dismissable
-    >
-        <span data-translate="pci_databases_general_information_no_ip"></span>
-    </oui-message>
-
-    <oui-message
-        data-ng-if="$ctrl.allowedIps.length === 0 && $ctrl.users.length === 0"
-        data-type="warning"
-        data-dismissable
-    >
-        <span
-            data-translate="pci_databases_general_information_no_ip_no_users"
-        ></span>
-        <a href data-ng-click="$ctrl.manageUsers()">
-            <span
-                data-translate="pci_databases_general_information_manage_users"
-            ></span>
-            <span
-                class="oui-icon oui-icon-arrow-right ml-1"
-                aria-hidden="true"
-            ></span>
-        </a>
+        <span data-translate="{{$ctrl.getWarningMessage().message}}"></span>
+        <button
+            class="oui-link"
+            type="button"
+            data-ng-if="$ctrl.getWarningMessage().userLink"
+            data-ng-click="$ctrl.manageUsers()"
+            data-translate="{{$ctrl.getWarningMessage().linkMessage}}"
+        ></button>
```

Comment:
`$ctrl.getWarningMessage()` will be called multiple times here. Can you store its result once, and use the value here?

Reply:
The result can change with the pooling. If I store the value this would not reflect the changes